src/components/dictionaryConcepts/components/MappingModal.jsx

```diff
@@ -45,17 +45,6 @@ export class MappingModal extends Component {
     });
   };
 
-  EditMapping = () => {
```

Comment:
Why delete this? Is related to the ticket?

Reply:
Yes. It is related to the ticket